_posts/2016-04-14-using-bumbler-to-reduce-runtime-dependencies.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Using Bumbler to reduce runtime dependencies"
+date: 2016-04-14 11:53:00
+categories: ["ruby", "rails", "performance"]
+author: "mauro-oto"
+---
+
+A few weeks ago, I ran into an interesting project called
+ [Bumbler](https://github.com/nevir/Bumbler). Bumbler lets you
+ know which are your Bundler-based project's largest dependencies.
```

Comment:
@mauro-oto If you split this sentence in too it will be easier to follow. Something like this: 

Bumbler shows you your project's largest dependencies. You project must use a `Gemfile` (bundler)

or

If your project uses Bundler, Bumbler shows your project's largest dependencies.